source/slang/slang-emit-c-like.cpp

```diff
@@ -2610,6 +2610,10 @@ void CLikeSourceEmitter::defaultEmitInstExpr(IRInst* inst, const EmitOpInfo& inO
         }
         break;
     }
+    case kIROp_RequireGLSLExtension:
+    {
+        break; //should already have set requirement; case covered for empty intrinsic block
+    }
```

Comment:
correct; I encountered the same problem now and since I understand what happens internally now I decided to fix the issue~ it was a simple problem of: kIROp_RequireGLSLExtension is not always possible to remove early (and emitter was not set to handle the op). Since this intrinsic was only used twice before this issue went unnoticed.